inlong-sort/sort-connectors/postgres-cdc/pom.xml

```diff
@@ -138,6 +138,10 @@
                                     <pattern>org.apache.inlong.sort.base</pattern>
                                     <shadedPattern>org.apache.inlong.sort.cdc.postgres.shaded.org.apache.inlong.sort.base</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>org.apache.inlong.sort.cdc.base</pattern>
+                                    <shadedPattern>org.apache.inlong.sort.cdc.postgres.shaded.org.apache.inlong.sort.cdc.base</shadedPattern>
```

Comment:
maybe other connector that use this cdc-base alse need to shade

Reply:
yes indeed, but ut only fails for postgres, I will add the relocation for other connectors